### PR TITLE
docs(hub) add route by cookie plugin to kong hub

### PR DESCRIPTION
**Pull request by redhoyasa**
- Adding Route by Cookie plugin to Kong Hub

### CODE REVIEW

File: app/_hub/redhoyasa/route-by-cookie/index.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+name: Route By Cookie
+publisher: redhoyasa
+categories:
+  - traffic-control
+type: plugin
+desc: Routing user request based on request cookies.
+description: |
+    Kong has several balancing algorithm options to forward user's request. Even though users will be routed consistently by Kong, but we cannot determine which user go to the certain host. This plugin will route user requests based on request cookies in deterministic way.
```
Comment (levfishbluefish) [inline app/_hub/redhoyasa/route-by-cookie/index.md]:
```suggestion
    Kong has several balancing algorithm options to forward user's request. Even though users will be routed consistently by Kong, we cannot determine which users will go to the certain host. This plugin will route user requests based on request cookies in deterministic way.
```

File: app/_hub/redhoyasa/route-by-cookie/index.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+$ curl -X POST http://localhost:8001/services/appService/plugins \
+    --data "name=route-by-cookie"  \
+    --data "config.target_upstream=v2.app.com" \
+    --data "config.cookie_name=APP_VERSION" \
+    --data "config.cookie_val=2"
+...
+{"created_at":1570980190, ... ,"id":"f3964aa9-bd3f-4044-94cb-49259e3e2349"}
+```
+
+Now any user with cookie `APP_VERSION=2` will be routed to `v2.app.com`. If the cookie value does not match with the cookie value defined in config, the request would not forwarded to the target upstream.
```
Comment (levfishbluefish) [inline app/_hub/redhoyasa/route-by-cookie/index.md]:
```suggestion
Now any user with cookie `APP_VERSION=2` will be routed to `v2.app.com`. If the cookie value does not match the cookie value defined in config, the request will not be forwarded to the target upstream.
```

**Comment by redhoyasa** [comment]
Hi @levfishbluefish, thanks for the review. I've applied the suggestions.

Please tell me if anything else needs to be changed.

**Comment by GayleNeumann** [comment]
@redhoyasa Thank you for the plugin submission for our Kong Hub. Unfortunately, we are not accepting new plugin entries at this time as we are in the process of revising our Plugin Hub and plugin submission criteria. We will have updates to our contributing guidelines soon and ask that you check back in a month or so. Again, thank you for your submission as we appreciate your efforts and contributions.  
Best, 
Gayle Neumann
Documentation Manager

**Pull request closed without merging**